spark-common/src/main/java/com/k15t/spark/atlassian/AtlassianIframeContentServlet.java

```diff
@@ -56,7 +56,7 @@ protected String prepareIndexHtml(String indexHtml, RequestProperties props) thr
     /**
      * Callback that can be implemented by sub classes in order to modify the iframe content document, for example to inject information.
      */
-    protected void customizeIframeContentDocument(Document document) {
+    protected void customizeIframeContentDocument(Document document, RequestProperties props) {
```

Comment:
FYI: The lack of this parameter had caused some ugly workarounds involving ThreadLocals, so I decided to add it for version 3.